### PR TITLE
Mesh scripts: doc/typos/fixes/clarifications/simplifications

### PR DESCRIPTION
**Pull request by ldemailly**
```release-note
Improvements/corrections to the mesh expansion / vm setup script
```

### CODE REVIEW

File: install/tools/setupMeshEx.sh [hunk 1]
```diff
@@ -20,24 +20,27 @@
...
-# test them.
+# Currently the script include helpers for GKE, other providers will be added as
+# they are contributed and we test them.
 
 # Environment variables used:
 #
-# ISTIO_STAGING - directory where istio artifacts are downloaded, default to current dir
-# ISTIO_VERSION - istio version.
-# ISTIO_NAMESPACE - defaults to istio-system, needs to be set for custom deployments
+# ISTIO_STAGING - directory where istio artifacts are downloaded, default to
```
Comment (chxchx) [inline install/tools/setupMeshEx.sh]: ISTIO_STAGING is no longer being used given this PR. Perhaps remove this line.

Comment (ldemailly) [reply install/tools/setupMeshEx.sh]: thx ! still cleaning up as I go but good catch

File: install/tools/setupIstioVM.sh [hunk 2]
```diff
@@ -56,19 +73,6 @@ function istioInstall() {
 
   chown -R istio-proxy /etc/certs
```
Comment (ldemailly) [inline install/tools/setupIstioVM.sh]: package must be installed first so the istio-proxy user exists

File: install/tools/setupMeshEx.sh [hunk 3]
```diff
@@ -20,24 +20,25 @@
...
-# ISTIO_NAMESPACE - defaults to istio-system, needs to be set for custom deployments
+# ISTIO_NAMESPACE - control plane namespace, defaults to istio-system, only
+# needs to be set for custom deployments
 # K8S_CLUSTER - name of the K8S cluster.
-# ISTIO_VERSION - current version of istio
-# SERVICE_ACCOUNT - what account to provision on the VM. Defaults to istio.default.
-# SERVICE_NAMESPACE-  namespace where the service account and service are running. Defaults to
-#  the current workspace in kube config.
-
-# GCLOUD_OPTS - optional parameters for gcloud command, for example "--project P --zone Z".
```
Comment (ldemailly) [inline install/tools/setupMeshEx.sh]: called GCP_OPTS

**Pull request merged**